### DISCUSSION TITLE
Request for addition of tooltips on panel controls

### DISCUSSION THREAD

**Discussion by kaeland** [rabbibotton/clog#156] [Ideas]
@rabbibotton Hi Dr. Botton, thank you for building Clog! Yesterday was the first time I used Clog Builder, and I have to say that it is a refreshing way to build UIs! I'd like to spend more time learning how to use this, and in the process I may post feature suggestions or bugs here. I hope that's ok.  

As I am new to Clog Builder, one thing I noticed is that it is hard to tell what the controls/buttons at the top of the panel menu do. I'm wondering if [a small tooltip](https://getbootstrap.com/docs/4.0/components/tooltips/) that names each button may be applied to each control as you hover over the button. This may help the user, and be a quick way to tell what the control/button does. 

I've attached a video to convey what I've mentioned above. I'd love to know your thoughts on this, and please disregard if you have some other idea in mind. Just though I'd share. And again, clog looks amazing, and your enthusiasm for this great tool is infectious lol! Please be encouraged, and please keep up the great work too! :D 

https://user-images.githubusercontent.com/17867788/176800604-388d76ef-1890-4a0b-ba87-1061733ea60c.mov

**Comment by rabbibotton** [comment]
It happens to be exactly what I am working on :) There are tooltips in other places already, and will add to the bars soon. In the old days tooltips were automatic with the alt-text on imgs and turns out not any more, but will be there soon.

I also intend on adding some help and video guides, etc. For sure if what I add not enough let me know, I really appreciate all advice.

My long term goal is a tool that even a non techy can produce rich products with and techies all the more so.

**Reply by kaeland** [reply]
That's awesome!! Just made it through the chat app tutorial too!

**Comment by rabbibotton** [comment]
OK I couldn't wait :) Tool Tips done

**Reply by kaeland** [reply]
Thank you!! :D 

https://user-images.githubusercontent.com/17867788/176819424-f848841d-323f-44de-9ff8-34c0f83a02af.mov
